geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommandsDUnitTest.java

```diff
@@ -65,10 +71,10 @@
   public ClusterStartupRule rule = new ClusterStartupRule();
 
   @Rule
-  public GfshCommandRule gfsh = new GfshCommandRule();
+  public transient GfshCommandRule gfsh = new GfshCommandRule();
```

Comment:
Yes, exactly. And I changed DiskStoreCommandsDUnitTest to be Serializable so it serializes when the lambdas are executed. Otherwise an exception like this is thrown:
```
org.apache.geode.test.dunit.RMIException: While invoking org.apache.geode.management.internal.cli.commands.DiskStoreCommandsDUnitTest$$Lambda$53/48305285.run in VM 1 running on Host localhost with 4 VMs
	at org.apache.geode.test.dunit.VM.execute(VM.java:537)
	at org.apache.geode.test.dunit.VM.invoke(VM.java:425)
	at org.apache.geode.test.dunit.VM.invoke(VM.java:401)
	at org.apache.geode.test.dunit.VM.invoke(VM.java:360)
	at org.apache.geode.test.junit.rules.VMProvider.invoke(VMProvider.java:85)
	at org.apache.geode.management.internal.cli.commands.DiskStoreCommandsDUnitTest.validateDiskStoreDiskDirectoryPath(DiskStoreCommandsDUnitTest.java:488)
Caused by: java.rmi.MarshalException: error marshalling arguments; nested exception is: 
	java.io.NotSerializableException: org.apache.geode.management.internal.cli.commands.DiskStoreCommandsDUnitTest
	at sun.rmi.server.UnicastRef.invoke(UnicastRef.java:158)
	at java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:227)
	at java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteObjectInvocationHandler.java:179)
	at com.sun.proxy.$Proxy32.executeMethodOnObject(Unknown Source)
	at org.apache.geode.test.dunit.VM.execute(VM.java:534)
```